forge-gui/res/cardsfolder/upcoming/skywriter_djinn.txt

```diff
@@ -0,0 +1,13 @@
+Name:Skywriter Djinn
+ManaCost:3 U
+Types:Creature Djinn
+PT:3/4
+K:Flying
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigMakeCard | TriggerDescription$ When CARDNAME enters the battlefield conjure a random card from CARDNAME's spellbook into your hand. Then if you have fewer cards in hand than the number of basic land types among lands you control, repeat this process.
+SVar:TrigMakeCard:DB$ MakeCard | Conjure$ True | AtRandom$ True | Spellbook$ Hard Evidence,Stifle,Contentious Plan,Suit Up,Befuddle,Majestic Metamorphosis,Hampering Snare,Ghostform,See the Truth,Keep Safe,Teferi's Time Twist,Startle,Flood of Recollection,Choking Tethers,Bury in Books | Zone$ Hand | SubAbility$ DBRepeat 
+SVar:DBRepeat:DB$ Repeat | RepeatCheckSVar$ X | RepeatSVarCompare$ LTY | ConditionCheckSVar$ X | ConditionSVarCompare$ LTY | RepeatSubAbility$ DBMakeCard
```

Comment:
does this need both checks?

Reply:
Yeah, otherwise you always get at least 2
![image](https://user-images.githubusercontent.com/67333662/193659035-211a9f43-7973-4d7f-96ff-4eccb618846c.png)